### PR TITLE
Problem: Makefile.am searches for header files of private classes in …

### PR DESCRIPTION
**Pull request by jana-rapava**
…include/.

Solution: Make the class public; regen.

Signed-off-by: Jana Rapava <janarapava@eaton.com>

### CODE REVIEW

**Comment by EldoreiJK** [comment]
Travis complained about clearing the files, so I tried to update within #6 and will see the results.

File: include/fty_warranty.h [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
-    it under the terms of the GNU General Public License as published by
-    the Free Software Foundation; either version 2 of the License, or
-    (at your option) any later version.
-
-    This program is distributed in the hope that it will be useful,
-    but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-    GNU General Public License for more details.
-
+    Copyright (C) 2014 - 2017 Eaton
```
Comment (jimklimov) [inline include/fty_warranty.h:4]: Old `gsl` in use, apparently. May impact other generated code nuances.
Did you compare the actual changeset before merging?

Remember that `git difftool -y` if your friend with whatever backend it uses to compare (I like `meld`).

File: .gitignore [hunk 2]
```diff
@@ -141,7 +141,6 @@ Findlibsodium.cmake
 Findlibzmq.cmake
 Findmalamute.cmake
 Findtntdb.cmake
-builds/cmake/*
```
Comment (jimklimov) [inline .gitignore:144]: Why did this ignorance disappear?

File: builds/cmake/ci_build.sh [hunk 3]
```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/env bash
```
Comment (jimklimov) [inline builds/cmake/ci_build.sh:1]: Why did this file appear? We don't do cmake here :)

File: builds/cmake/ci_build.sh [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+else
+    PKG_CONFIG_PATH=${BUILD_PREFIX}/lib/pkgconfig $CI_TIME cmake "${CMAKE_OPTS[@]}" .
+    $CI_TIME make all VERBOSE=1 -j4
+    $CI_TIME ctest -V
+    $CI_TIME make install
+fi
+[ -z "$CI_TIME" ] || echo "`date`: Builds completed without fatal errors!"
+
+echo "=== Are GitIgnores good after making the project '$BUILD_TYPE'? (should have no output below)"
+git status -s || true
```
Comment (jimklimov) [inline builds/cmake/ci_build.sh:263]: This line exposes a non-recent zproject (there is a `make` target for such sanity check now)

**Pull request merged**